README.md

```diff
@@ -85,44 +85,46 @@ specified below each.
-| encode                                                                        |     39,041 |     31,769 |     76,403 |     16,615 |     16,910 |
-| decode                                                                        |     25,185 |     24,287 |     34,437 |     32,388 |     27,999 |
-| Medium complex object                                                         |            |            |            |            |            |
-| encode                                                                        |     10,382 |     11,427 |     32,995 |      3,959 |      5,275 |
-| decode                                                                        |      9,785 |      9,796 |     11,515 |      5,898 |      7,200 |
-| Array with 256 True values                                                    |            |            |            |            |            |
-| encode                                                                        |    114,341 |    101,039 |    344,256 |     62,382 |     72,872 |
-| decode                                                                        |    149,367 |    151,615 |    181,123 |    114,597 |    130,392 |
-| Array with 256 dict{string, int} pairs                                        |            |            |            |            |            |
-| encode                                                                        |     13,715 |     14,420 |     51,942 |      3,271 |      6,584 |
-| decode                                                                        |     12,670 |     11,788 |     12,176 |      6,743 |      8,278 |
-| Dict with 256 arrays with 256 dict{string, int} pairs                         |            |            |            |            |            |
-| encode                                                                        |         50 |         54 |        216 |         10 |         23 |
-| decode                                                                        |         32 |         32 |         30 |         20 |         23 |
-| Dict with 256 arrays with 256 dict{string, int} pairs, outputting sorted keys |            |            |            |            |            |
-| encode                                                                        |         46 |         41 |            |          8 |         24 |
-| Complex object                                                                |            |            |            |            |            |
-| encode                                                                        |        533 |        582 |            |        408 |        431 |
-| decode                                                                        |        466 |        454 |            |        154 |        164 |
+- CPython 3.11.3 (main, Apr  6 2023, 07:55:46) [GCC 11.3.0]
+- ujson        : 1.36.dev558
```

Comment:
Seems like it's using the v1.35 tag and ignoring the higher version tags without the `v` prefix.

Reply:
Aha, it's because I ran it on my branch and the tags weren't in sync with upstream.

I'll redo it, and also use `run: git fetch --prune --unshallow`, which is what we have for `deploy.yml`

Reply:
https://github.com/hugovk/ultrajson/actions/runs/5148581127/jobs/9270556867 running with `ujson-5.7.1.dev26`, will update the benchmark stats here tomorrow 👍

Reply:
Updated!